### ISSUE TITLE
bower.json is not valid JSON

### ISSUE THREAD

**Issue by yuriiz** [zurb/foundation-sites#11364]
https://github.com/zurb/foundation-sites/blob/develop/bower.json in develop branch as of today is not a valid JSON file. There is a syntax error at line #19:
```
"authors": [
    "ZURB <foundation@zurb.com> (http://foundation.zurb.com)",
],
```
Unlike Javascript, in JSON there can be no comma before `]`.

**Comment by DanielRuf** [comment]
Hi @yuriiz,

thanks for reporting this.

Did this break the install for you?

**Comment by DanielRuf** [comment]
Please see https://github.com/zurb/foundation-sites/blob/support/6.5/bower.json, so this is already addressed in 6.5.0 RC1

**Comment by yuriiz** [comment]
https://github.com/zurb/foundation-sites/blob/support/6.5/bower.json is 10 days ago.
https://github.com/zurb/foundation-sites/blob/develop/bower.json is 5 days ago.
So I thought it was broken. I'm not sure which version is going to be used in future but it needs to be not the one with error.

**Comment by ncoden** [comment]
Sorry, my bad. My JS lint configuration require a leading `,` in arrays and objects. I often make this error when working on JSON files.

Fixed in https://github.com/zurb/foundation-sites/pull/11367

**Comment by DanielRuf** [comment]
> Please see /6.5/bower.json@support, so this is already addressed in 6.5.0 RC1

Wasn't this already done in support/6.5 @ncoden? And I guess you mean trailing.
